hw1/02-palindrome.js

```diff
@@ -1,3 +1,41 @@
-const elem = document.querySelector('input');
+// Declarations
+const elem = document.querySelector("input");
+elem.addEventListener("change", handleInput); // get input on enter
+const output = document.getElementById("result"); // output the result to HTML
 
-elem.addEventListener('input', handleInput);
+// Check for positive number
+function isPositiveNumber(num) {
+  if (num < 0) {
+    return false;
```

Comment:
Same thing I said to a few others - make sure you're checking that `Number(num) < 0` also be sure to return true outside of this statement as well.

Reply:
You mean, like cast the num variable to a Number to ensure that I'm working with a number? That's a good observation!